test/runner.py

```diff
@@ -488,6 +488,7 @@ def parse_args():
                            'Useful when combined with --failfast')
   parser.add_argument('--force64', action='store_true')
   parser.add_argument('--crossplatform-only', action='store_true')
+  parser.add_argument('--force-browser-process-termination', action='store_true', help='If true, a fail-safe method is used to ensure that all browser processes are terminated before and after the test suite run.')
```

Comment:
Is there any reason not to just do this on startup?   

Seems like doing that way would also make it easier to debug any stalled processes and doing on shutdown doesn't really help any use cases does it?

Reply:
The CI might run other non-browser suites after browser suites.. so if there were stray browser processes, they would persist over to hogging resources in those next suites.

Debuggability is a non-concern.. when debugging, one will naturally just not pass this flag. This is intended specifically for unattended production runs, where one is not there to debug anything.

Reply:
I was think it might also be useful for my workflow which is:

1. Always over ssh, so always headless.
2. On a host where I never expect to have existing browsers running.
3. I find myself often running `pkill -f google-chrome` , "just in case" before I try to run any browser tests.

Reply:
Would it be useful to split to two flags, `--force-browser-terminate-before` and `-after`?

Reply:
No worries the `CI might run other non-browser suites after browser suites` rational makes sense to me.

Do you want to make this an env var too so you can apply it universally to your CI bots?